src/checkers/inference/solver/backend/lingeling/LingelingSolver.java

```diff
@@ -33,7 +33,10 @@
 public class LingelingSolver extends MaxSatSolver {
 
     // Lingeling binary executable file should be located at JSR308/lingeling/lingeling.
-    private final String lingeling = System.getenv().get("JSR308") + "/lingeling/lingeling";
+    private String currentDirPath = System.getProperty("user.dir");
```

Comment:
user.dir depends on which directory "java" command is executed from (ie the inference script), it seems at a glance that the code would only work if executed within the CFI subdirectories. Can you give it a test by running the inference script with your bash shell set to root "/" or your user home "~" directory?

Reply:
Sure, no problem, I will test it tonight and let you know as soon as possible.

Reply:
I tested the inference script and it worked. I post the test command and logs [here](https://paste.ubuntu.com/p/VzDxn4DwHk/).